src/main/kotlin/uk/co/craigbass/pratura/math/CurrencyExtensions.kt

```diff
@@ -4,3 +4,5 @@ import java.math.BigDecimal
 
 fun String.toDecimal(): BigDecimal = BigDecimal(this)
 fun Int.toDecimal(): BigDecimal = BigDecimal(this)
+fun BigDecimal?.toCurrencyWithSymbol() = if (this == null) "£0.00" else "£${toCurrencyString()}"
```

Comment:
generally disagree with extension methods on built in types. They are basically global functions that are dressed up to seem cool. Generally would avoid unless it's really obvious it would be used everywhere, or you are inside tests. Like ```String.toDecimal``` massive possibilities about what that could do. e.g. how does it handle commas, periods, extra non numeric characters etc.